### PR TITLE
(DOCSP-26570): Fixed build errors.

### PR DESCRIPTION
**Pull request by corryroot**
@ccho-mongodb, I'm down to one build error that I can't figure out.

[BUILD](https://workerpool-boxgs.mongodbstitch.com/pages/job.html?collName=queue&jobId=637d110fed8015c14bcc3f1f)

### CODE REVIEW

File: source/style-guide/markup/headings.txt [hunk 1]
```diff
@@ -73,19 +73,19 @@ To avoid errors in your header markup syntax, stick to the following rules:
             ----------------
 
      * -
-         This example assumes the source constant ``{+Db+}`` resolves to 
+         This example assumes the source constant ``{{+db-example+}}`` resolves to
```
Comment (ccho-mongodb) [inline source/style-guide/markup/headings.txt:76]: Cool, didn't you could format source constants like this!

Comment (ccho-mongodb) [reply source/style-guide/markup/headings.txt:76]: Strange that this page doesn't seem to exist in the left nav. I guess we should add ensuring the pages are all surfaced in the navigation to our list of tasks.

File: source/images-guide.txt [hunk 2]
```diff
@@ -61,13 +61,13 @@ itself should not be wider than 1000px.
    content.
 
    It may make sense to use
-   `mozjpeg <https://github.com/mozilla/mozjpeg>`_ if you need lossy
-   compression; `imagemagick <http://imagemagick.org/>`_ to resize images;
-   and a `Ninja <http://ninja-build.org/>`_ file to put the pieces
+   `mozjpeg <https://github.com/mozilla/mozjpeg>__ if you need lossy
```
Comment (ccho-mongodb) [inline source/images-guide.txt:64]: Issue: I think this is missing a closing backtick between the angled bracket and underscores.

File: source/style-guide.txt [hunk 3]
```diff
@@ -64,11 +61,11 @@ MongoDB external and internal customers.
 The |mdbsg| includes material from the following:
 
 - `Google Developer Documentation Style Guide <https://developers.google.com/style/>`__
-  by `Google <https://www.google.com>`__, licensed under
-  :cc:`CC BY 4.0 </by/4.0/>`;
+  by `Google <https://www.google.com>`__, licensed under `CC BY 4.0
```
Comment (ccho-mongodb) [inline source/style-guide.txt:64]: Should this also use double-underscore or was a single one intentional? I think when using a single one, [you can separate the definition and avoid repetition](https://sublime-and-sphinx-guide.readthedocs.io/en/latest/references.html#links-to-external-web-pages), but I don't think we use that pattern often.

File: source/style-guide/style/keyboard-keys.txt [hunk 4]
```diff
@@ -98,4 +96,4 @@ combinations, names, and shortcuts, see :ref:`text-formatting`.
      - Press :kbd:`F3` to find the next matching process, or press
        :kbd:`Esc` to quit the search.
 
-       Press the Help key (`F1`).
+       Press the Help key (``F1``).
```
Comment (ccho-mongodb) [inline source/style-guide/style/keyboard-keys.txt:99]: Should thisi be ":kbd:" instead of monospace?

**Review by ccho-mongodb — APPROVED** [review]
LGTM, just a few small items and questions.

File: source/images-guide.txt (hunk 2 above)
Comment (corryroot) [reply source/images-guide.txt:64]: Good catch!

File: source/style-guide.txt (hunk 3 above)
Comment (corryroot) [reply source/style-guide.txt:64]: Another good catch!

File: source/style-guide/style/keyboard-keys.txt (hunk 4 above)
Comment (corryroot) [reply source/style-guide/style/keyboard-keys.txt:99]: Yes!

**Pull request merged**